qiita_core/support_files/config_test.txt

```diff
@@ -36,6 +36,9 @@ BASE_DATA_DIR =
 # Valid upload extension, comma separated. Empty for no uploads
 VALID_UPLOAD_EXTENSION = fastq,fastq.gz,txt,tsv,sff,fna,qual
 
+#Portal the site is working under
+PORTAL = QIIME_EMP
```

Comment:
We currently have this values:
```
 portal_type_id |  portal   |  portal_description   
----------------+-----------+-----------------------
              1 | QIIME     | QIIME portal
              2 | EMP       | EMP portal
              3 | QIIME_EMP | QIIME and EMP portals
```

I would suggest changing these to:
```
 portal_type_id |  portal   |  portal_description   
----------------+-----------+-----------------------
              1 | QIITA     | QIITA portal
              2 | EMP       | EMP portal
```

Note that we don't need a `QIITA_EMP` since QIITA has access to everything. The first one `QIITA` is used to represent does studies that do not belong to any other portal.